socialcards/src/CardMaker.jsx

```diff
@@ -39,38 +39,81 @@ const CardMaker = ({ token }) => {
   ];
 
   console.log(token);
-  const handleSubmit = () => {
-    axios.post(
-      "https://social-cards.fly.dev/api/cards/",
-      {
-        styles: [
-          { property: "backgroundColor", value: `${coverBackgroundColor}` },
-          { property: "color", value: `${coverTextColor}` },
-          { property: "textShadow", value: `${addTextShadow}` },
-          { property: "textAlign", value: `${textOrientation}` },
-          {
-            property: "border",
-            value: `${coverBorderWidth} ${coverBorderStyle} ${coverBorderColor}`,
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    axios
+      .post(
+        "https://social-cards.fly.dev/api/cards/",
+        {
+          front_text: coverText,
+          back_text: insideText,
+          image_url: selectedImage,
+        },
+        {
+          headers: {
+            Authorization: `Token ${token}`,
           },
-          // { property: "", value: `${}` },
-          // { property: "", value: `${}` },
-          // { property: "", value: `${}` },
-          // { property: "", value: `${}` },
-          // { property: "", value: `${}` },
-          // { property: "", value: `${}` },
-        ],
-        front_text: coverText,
-        back_text: insideText,
-        imageURL: `${selectedImage}`,
-        font: selectedFont,
-        draft: isDraft,
-      },
-
-      {
-        headers: { Authorization: `Token ${token}` },
-      }
-    );
+        }
+      )
+      .then((res) => {
+        return axios
+          .request({
+            url: `https://social-cards.fly.dev/api/cards/${res.data.id}/styles/`,
+            method: "POST",
+            headers: {
+              Authorization: `Token ${token}`,
+            },
+            data: [
+              {
+                property: "backgroundColor",
+                value: coverBackgroundColor,
+              },
+              {
+                property: "fontFamily",
+                value: selectedFont,
+              },
+              {
+                property: "textAlign",
+                value: textOrientation,
+              },
+            ],
+          })
+          .then((res) => console.log(res.data));
```

Comment:
Once a user has succeeded in creating a card, you probably want to do something in the UI to indicate that the action was successful. Presumably once they have submitted, they aren't going to need to still be on the editor page. Maybe navigate to the show card view to see the finished card? You have the card id so that should be easy to do with the `useNavigate` hook from React Router. 

Also, another suggestion here: generally we want to include a `.catch()` block for a request to make sure we have a plan for what should happen if the request does not result in a successful response -- any response in a 400 or 500 range would end up in the catch block and we could handle it there -- like setting an error state and showing the user an error message (or messages) to let them know what went wrong and maybe directing them to fix errors and resubmit.

Reply:
Added!